rundeckapp/build.gradle

```diff
@@ -230,6 +230,7 @@ bootRun {
             '-Xmx2048m')
     systemProperties(System.properties)
     String springProfilesActive = 'spring.profiles.active'
+    systemProperty "rundeck.jaaslogin", "true"
```

Comment:
Was this put in for testing? I'm not sure we want jaaslogin to be default.

Reply:
I need to double check but I believe it wouldn't pick up the service provider at all without jaas login engabled(I'm not sure why this is or how it works at all without it enabled).

JAAS login is on by default in our Docker, deb, and rpm distributions .